### DISCUSSION TITLE
QR code generator for wallet.cashu.me

### DISCUSSION THREAD

**Discussion by llamabyte** [cashubtc/nutshell#273] [Ideas]
Would be great to have the ability to create a QR code within the wallet that if scanned, sends you to redeem or wallet.

![Untitled](https://github.com/cashubtc/cashu/assets/130471564/d66c1177-435d-4f0b-89c3-4b4d2f3cf05d)

**Comment by aurorabor** [comment]
This is already up for discussion ([https://github.com/cashubtc/cashu.me/issues/44](https://github.com/cashubtc/cashu.me/issues/44)), but not trivial since ecash Token are relatively information dense and there is no standardized way in displaying these long strings.

**Reply by callebtc** [reply]
Yeah, the wallet can already show QR's but only if you're sending one token currently... :)

That means, it would work for tokens of amounts 2, 4, 8, 16, 32, etc.... To show more than one token, we have to come up with an animated QR standard as @aurorabor mentioned.

**Comment by zouxianwei1984** [comment]
any discuss group?
